modules/nf-core/ashlar/meta.yml

```diff
@@ -0,0 +1,45 @@
+name: "ashlar"
+description: Alignment by Simultaneous Harmonization of Layer/Adjacency Registration
+keywords:
+  - image_processing
+  - alignment
+  - registration
+tools:
+  #
+  - "ashlar":
+      description: "Alignment by Simultaneous Harmonization of Layer/Adjacency Registration"
+      homepage: "https://labsyspharm.github.io/ashlar/"
+      documentation: "https://labsyspharm.github.io/ashlar/"
+      doi: "10.1093/bioinformatics/btac544"
+      licence: "['MIT']"
+
+input:
+  #
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', : args: '--flip-x' ]
+  - file_in:
+      type: file
+      description: multidimentional image data in formats from most commercial microscopes
+      pattern: "*.{ome.tiff,ome.tif,rcpnl,btf,nd2,tiff,tif,czi}"
+output:
+  #
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', : args: '--flip-x' ]
```

Comment:
Delete `args` from example

Reply:
oops, yes, I forgot this template was also in meta.yml and would need to be updated.  thank you for catching it!